examples/ota-requestor-app/linux/main.cpp

```diff
@@ -55,15 +55,10 @@ using namespace chip::ArgParser;
 using namespace chip::Messaging;
 using namespace chip::app::Clusters::OtaSoftwareUpdateProvider::Commands;
 
-constexpr size_t kMaxActiveCaseClients = 2;
-constexpr size_t kMaxActiveDevices     = 8;
-
 OTARequestor gRequestorCore;
 LinuxOTARequestorDriver gRequestorUser;
 BDXDownloader gDownloader;
 LinuxOTAImageProcessor gImageProcessor;
-CASEClientPool<kMaxActiveCaseClients> gCASEClientPool;
-OperationalDeviceProxyPool<kMaxActiveDevices> gDevicePool;
```

Comment:
Since these aren't needed anymore, can you please remove the includes on like 19-20